src/test/kotlin/FetchBlockIdCommandTest.kt

```diff
@@ -0,0 +1,24 @@
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+class FetchBlockIdCommandTest {
+    private lateinit var command: FetchBlockIdCommand
+
+    @BeforeEach
+    fun setUp() {
+        command = FetchBlockIdCommand()
+    }
+
+    private val startHeight = 730000
+
+    @Test
+    fun `should fetch block id`() = runBlocking {
+        val expectedBlockId = "0000000000000000000384f28cb3b9cf4377a39cfd6c29ae9466951de38c0529"
+        command.startHeight = startHeight
+        val blockId = command.execute()
+
+        assertEquals(expectedBlockId, blockId)
+    }
+}
```

Comment:
This unit test is not working even though logic seems good.
I've found out that my command classes print the results directly to the console.

- They do not return any values which can be tested. 
- Since I'm directly executing the command to the console, I find it hard to test.
- Shall I make changes to my business logic? And instead of directly executing, I can return a string maybe.

Are there any alternative solutions to this problem.
And Is this approach correct, or am I missing something?